roles/signals.py

```diff
@@ -0,0 +1,18 @@
+"""
+Signals handlers
+"""
+from __future__ import unicode_literals
+
+from django.db.models.signals import post_migrate
+from django.dispatch import receiver
+
+from roles.utils import sync_groups_permissions
+
+
+# pylint: disable=unused-argument
+@receiver(post_migrate)
+def my_callback(sender, **kwargs):
```

Comment:
Is it possible to use this to determine what app is sending the migration?  If so, it might be nice to only run this for those migrations instead of churning the DB on every single one.  If you decide that isn't worth it, then just change the args to **kwargs since there is no real reason to handle `sender` (or *args, **kwargs if one of them is a pos arg).